src/styles/card-block.scss

```diff
@@ -0,0 +1,116 @@
+@import "./stars.scss";
+
+@import "./variables.scss";
```

Comment:
Import variables in main file before other blocks files, no need to import it in every file